lib/menu.rb

```diff
@@ -0,0 +1,12 @@
+class Menu
+
+  attr_reader :menu
+
+  def initialize
+    @menu = [{dish: "cianfrusaglie alla rinfusa", price: 3}, {dish: "raudi accesi", price: 3}, {dish: "fragole alla carbonara", price: 14}]
+  end
+
+  def entry(dish = Dish.new)
```

Comment:
The default here will cause an argument error as `Dish.new` requires 2 parameters.